test/gpu/native/multiGPU/worksharingBasic.chpl

```diff
@@ -0,0 +1,26 @@
+config const n = 1024;
+
+writeln("Number of sublocales: ", here.getChildCount());
+
+var A: [0..#n] int;
+// assign half the work to CPU, the rest to GPUs. Assume divisibility
+const numGPUs = here.getChildCount()-1;
+const cpuSize = n/2; 
+const gpuSize = (n/2)/numGPUs;
+
+assert(n%2 == 0);
+assert((n/2)%numGPUs == 0);
```

Comment:
This assumption worries me a little bit. I'd like to avoid making assertions about the underlying hardware as much as possible. If we moved this to another machine that had (say) 10 GPUs per node this would fail, right?

Maybe we could compute n by doing something like `n = numGpus * 1024`.

Reply:
I was too lazy to do that. If you think strongly, I can make the test stronger there.